CHANGELOG.rst

```diff
@@ -7,7 +7,6 @@ CHANGELOG
 
 * enhancement: Workflow: Specify tasks from which training/tuning operator to transform/deploy in related operators
 * feature: support for Tensorflow 1.12
```

Comment:
Remove support for TF 1.12 as well (repeated)

Reply:
done